content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -627,13 +631,84 @@ read-modify-write and/or patch are the following:
 It is strongly recommended for controllers to always "force" conflicts, since they
 might not be able to resolve or act on these conflicts.
 
+### Transferring Ownership
+
+In addition to the concurrency controls provided by [conflict
+resolution](#conflicts), Server Side Apply provides ways to perform coordinated
+field ownership transfers from users to controllers.
+
+First, the user delegates ownership of the field to a temporary, uniquely named,
+field manager. Next, the user signals to the controller that it should take
+ownership of the field, usually by setting some other field (perhaps in some
+other object) that the controller observes. Last, controller then claims ownership
+of the field by updating the object or applying a configuration containing the
+field.
+
+Transferring fields to controllers this way ensures the field is not accidently
+removed from the live object, or reset to a default value, at any point of time
+during the transfer.
+
+This approach can be used, for example, to safely transfer ownership of the
+`replicas` field from a user to a controller while enabling Horizontal Pod
+Autoscaling on an object.
+
+Say a user has defined deployment with `replicas` set to 100:
+
+{{< codenew file="application/ssa/nginx-deployment.yaml" >}}
+
+And the user created the deployment using server side apply like so:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment.yaml --server-side
+```
+
+Before the user enables Horizontal Pod Autoscaling, the user needs to transfer
+the `replicas` field to a temporary field manager by defining a configuration
+that owns just the replicas field:
+
+{{< codenew file="application/ssa/nginx-deployment-replicas-only.yaml" >}}
+
+The user applies this configuration using a unique field manager name:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment-replicas-only.yaml --server-side --field-manager=unique-manager-1s03n45l4n --validate=false
+```
+
+Next, the user removes the `replicas` field from original configuration:
+
+{{< codenew file="application/ssa/nginx-deployment-no-replicas.yaml" >}}
+
+The user then applies this configuration:
+
+```shell
+kubectl apply -f application/ssa/nginx-deployment-no-replicas.yaml --server-side
+```
+
+Now the user may safely enable Horizontal Pod Autoscaling, e.g.:
```

Comment:
No, now it's safe to make other changes.

Reply:
Actually, it was always safe to make unrelated changes. It's removing the replicas field that was the potentially unsafe thing.

Reply:
Right. I've rephrased things so it's more clear that the replica field removal is the touchy part.